src/PullRequestsAPI.ts

```diff
@@ -48,7 +48,8 @@ class GithubPullRequestAPI implements PullRequestAPI {
       repo: this.repo,
       issue_number: issue_number,
       state: 'closed'
-    })
+    }).catch(error => console.log(`An error occurred closing the pull request #${issue_number}`, error))
```

Comment:
As I mentioned earlier I think we could just handle this in the for loop. If any of these fail, the following request are also fairly likely to fail.

Reply:
Fair point. Updated.